server/start.go

```diff
@@ -488,9 +495,66 @@ func startInProcess(svrCtx *server.Context, clientCtx client.Context, opts Start
 		return err
 	}
 
+	// Start running rpc server for sgx binary access on the evm Keeper statedb
+	listener, err := startRpcServer(svrCtx, g, app)
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		// TODO: Graceful stop
+		listener.Close()
+	}()
+
 	return g.Wait()
 }
 
+func startRpcServer(
+	svrCtx *server.Context,
+	g *errgroup.Group,
+	app types.Application,
+) (listener net.Listener, err error) {
+	ethApp := app.(IncoApp)
+	if ethApp == nil {
+		return nil, nil
+	}
+
+	evmKeeper := ethApp.GetEvmKeeper()
+	if evmKeeper == nil {
+		return nil, nil
+	}
+
+	g.Go(func() error {
+		listener, err = runRPCServer(svrCtx, evmKeeper)
+		return err
+	})
+
+	return
+}
+
+func runRPCServer(svrCtx *server.Context, keeper *evmKeeper.Keeper) (net.Listener, error) {
+	defer func() {
+		if r := recover(); r != nil {
+			svrCtx.Logger.Error("recovered from panic", "error", r)
+		}
+	}()
+
+	// Run a persistent RPC server for sgx binary can access to evm keeper statedb
+	srv := &evmKeeper.EthmRpcServer{Keeper: keeper}
+	rpc.Register(srv)
```

Comment:
```suggestion
	err := rpc.Register(srv)
```

if there's an error, let's return it

Reply:
Agree, done.